.github/workflows/cncf-conformance.yaml

```diff
@@ -0,0 +1,146 @@
+name: cncf-conformance
+# Workflow to run CNCF conformance tests on MicroShift upstream latest release
+#
+# This workflow runs the CNCF conformance test suite using Sonobuoy in 'certified-conformance'
+# mode for official Kubernetes conformance certification.
+
+on:
+  schedule:
+    - cron: '0 4 * * *'  # Daily at 04:00 UTC
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "latest"
+        description: MicroShift version to test (e.g., 4.21.0_ga9cd00b34_4.21.0_okd_scos.ec.5 or 'latest' for most recent release)
+        type: string
+      registry:
+        default: "ghcr.io/microshift-io"
+        description: Container registry to pull bootc images from
+        type: string
+      test-timeout:
+        default: "8400"
+        description: Sonobuoy test timeout in seconds (8400 = ~2.5 hours)
+        type: string
+
+env:
+  VERSION: ${{ github.event.inputs.version || 'latest' }}
+  REGISTRY: ${{ github.event.inputs.registry || 'ghcr.io/microshift-io' }}
+  TEST_TIMEOUT: ${{ github.event.inputs.test-timeout || '8400' }}
+
+jobs:
+  cncf-conformance:
+    strategy:
+      matrix:
+        runners: [ubuntu-24.04, ubuntu-24.04-arm]
+    name: Run CNCF conformance tests (${{ matrix.runners }})
+    runs-on: ${{ matrix.runners }}
+    steps:
+      - name: Check out MicroShift upstream repository
+        uses: actions/checkout@v4
+
+      - name: Detect CPU architecture
+        id: detect-cpu-arch
+        uses: ./.github/actions/arch
+
+      - name: Prepare the test environment
+        uses: ./.github/actions/prebuild
+
+      - name: Pull pre-built bootc image
+        shell: bash
+        run: |
+          set -euo pipefail
+
+          # Resolve version (use latest release if not specified)
+          if [ "${{ env.VERSION }}" = "latest" ]; then
+            VERSION=$(gh release list --limit 1 --json tagName --jq '.[0].tagName')
+            echo "Using latest release version: ${VERSION}"
+          else
+            VERSION="${{ env.VERSION }}"
+            echo "Using specified version: ${VERSION}"
+          fi
+
+          IMAGE="${{ env.REGISTRY }}/microshift"
+          TAG="${VERSION}-${{ steps.detect-cpu-arch.outputs.go_arch }}"
+
+          echo "Pulling ${IMAGE}:${TAG}"
+          sudo podman pull "${IMAGE}:${TAG}"
```

Comment:
Why do we need a platform tag? It's a manifest so we just pull `microshift:<version>`